src/main/java/frc/robot/subsystems/vision/Constants.java

```diff
@@ -1,8 +1,9 @@
 package frc.robot.subsystems.vision;
 
 public class Constants {
-    public static final double CAMERA_HEIGHT = 0;
-    public static final double CAMERA_PITCH = 0;
-    public static final double UPPER_CONE_TARGET11_X_DISTANCE = 0;
-    public static final double UPPER_CONE_TARGET11_Y_DISTANCE = 0;
+    public static final double CAMERA_HEIGHT = 1.1; //[m]
+    public static final double CAMERA_PITCH = -0.611; //[radian]
+    public static final double TARGET_HEIGHT = 0.65; //[m]
+    public static final double UPPER_CONE_TARGET11_X_DISTANCE = 0; //[m]
+    public static final double UPPER_CONE_TARGET11_Y_DISTANCE = 0; //[m]
```

Comment:
It's better to represent this information as [`Posed3d`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/geometry/Pose3d.html)s.